Postgres/scaleUpPostgres.ps1

```diff
@@ -0,0 +1,60 @@
+$resourceURI = Get-AutomationVariable -Name 'PostgresResourceURI'
+$tenantId = Get-AutomationVariable -Name 'TennantID'
+$clientId =  Get-AutomationVariable -Name 'ClientId'
+$clientSecret =  Get-AutomationVariable -Name 'ClientSecret'
+
+$UnsecurePassword = $clientSecret
```

Comment:
Same as before- can we just use $clientSecret

Reply:
```suggestion
```